hbase-protocol-shaded/src/main/protobuf/Admin.proto

```diff
@@ -80,6 +80,8 @@ message OpenRegionRequest {
   repeated RegionOpenInfo open_info = 1;
   // the intended server for this RPC.
   optional uint64 serverStartCode = 2;
+  // Master active time as fencing token
+  optional int64 initiating_master_active_time = 3;
```

Comment:
There are some duplicate file under hbase-protocol module also? Do we need to update there as well ?